Extensions/move-blog-selector.js

```diff
@@ -0,0 +1,51 @@
+//* TITLE Blog Selector to bottom **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Moves the blog selector from top of form to bottom of form	**//
+//* DEVELOPER manavortex **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.moveblogselector = new Object({
+	
+	running: false,
+	
+	run: function() {
+		this.running = true;
+		this.waitForAndPrepend(".post-form--header", ".post-form--bottom");
+	},
+	
+	destroy: function() {
+		this.running = false;
+	}, 
+	
+	/**		
+	* Async function: call with checkElement('#myElement').then((el) { ... }) .
+	*
+	* @param {string} selector - document query selector.
+	*/
+	checkElement: async function(selector) {
+		while ( this.running && document.querySelector(selector) === null) {
+			await new Promise( resolve =>  requestAnimationFrame(resolve) );
```

Comment:
and here
```suggestion
			await new Promise(resolve =>  requestAnimationFrame(resolve));
```

Reply:
Also, requestAnimationFrame is too fast of an inner loop here—try a 100 millisecond setTimeout instead?